lib/tasks/reports.rake

```diff
@@ -0,0 +1,32 @@
+require 'csv'
+require 'date'
+
+namespace :reports do
+  desc 'creates a csv of all submissions for a grant'
+  task :grant_submissions, [:grant] => [:environment] do |t, args|
+    grant = Grant.where(name: args[:grant]).first
```

Comment:
You may want to includes other associations(submissions, questions, submission responses) here to prevent n+1 queries and to speed up rake tasks.